flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -918,6 +915,18 @@ private ApplicationReport startAppMaster(
                                 : Path.CUR_DIR,
                         LocalResourceType.FILE);
 
+        // usrlib will be automatically shipped if it exists.
+        final Set<File> usrLibShipFiles = new HashSet<>();
```

Comment:
Hmm. Maybe I did not make myself clear. I do not mean to remove the condition check.

```
if (ClusterEntrypointUtils.tryFindUserLibDirectory().isPresent()) {
    ... ...
}
```

Reply:
I remove this check because I double checked the codes and it should work well as we will do this check in addUsrLibFolderToShipFiles. 
For readability and less object allocation/function call , I agree to get it back.